release-tools/scripts/version-info.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+###### Information ############################################################################
+# Name:          version-info.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         Print the ES/ODFE related information and values
+#                as defined in the 'manifest.yml' file
+#
+# Usage:         ./version-info.sh $VERSION_TYPE
+#                $VERSION_TYPE (required): --od | --od-prev | --od-next
+#                                          --es | --es-prev | --es-next
+#
+# Requirements:  Need to install yq (v4.0.0+) on your system:
+#                https://github.com/mikefarah/yq/releases/
+#
+# Starting Date: 2021-01-06
+# Modified Date: 2021-01-06
+###############################################################################################
+
+set -e
+ROOT=`dirname $(realpath $0)`;
+MANIFEST_FILE="$ROOT/manifest.yml"
```

Comment:
Can you change MANIFEST_FILE="./opendistro-build/release-tools/scripts/manifest.yml" , that will help us avoid realpath  command. This is in regard to running this script on the macos instances where the realpath is not present by default.

Reply:
The idea is if you move the scripts folder these script will still work, as long as they are in same folder
Hardcode it defeat the purpose here